Tests/.swiftlint.yml

```diff
@@ -0,0 +1,29 @@
+
+disabled_rules: # rule identifiers to exclude from running
+  - closure_parameter_position
+  - empty_parentheses_with_trailing_closure
+  - force_try
+  - function_body_length
+  - opening_brace
+  - redundant_string_enum_value
+  - todo
+  - type_body_length
+
+file_length:
+  warning: 3000
+  error: 4000
+
+function_body_length:
+  warning: 50
+  error: 80
+
+line_length:
+  warning: 500
+  error: 999
+
+trailing_comma:
+  mandatory_comma: true
+
+trailing_whitespace:
+  severity: error
+
```

Comment:
I think it is reasonable to be more lenient in coding style / linting for tests as opposed to the source for the SDKs.  The only way I know to do that is have two separate files.  And I prefer to put this in a config file, rather than as exceptions in individual files, to show that this is acceptable style for tests.